benchmark/single-source/ObjectiveCBridging.swift

```diff
@@ -744,3 +747,29 @@ public func run_ASCIIStringFromFile(_ N: Int) {
   #endif
 }
 
+var UnicodeStringFromCodable:String? = nil
+var UnicodeStringFromCodableDict = [String:Void]()
```

Comment:
nitpick: why uppercase?

Reply:
tbh only because a nearby thing was, but I think I wrote that too, so the same question applies. I don’t know the answer >.<